CONTRIBUTING.md

```diff
@@ -0,0 +1,41 @@
+# Contributing to ExaDG
+We are happy that you wish to contribute to ExaDG. 
+Here you will find some useful information you need before you start developing. 
```

Comment:
"Here you will find some useful information you need before you start developing. "

suggestion for alternative formulation: "Below, you find information explaining how to contribute to ExaDG."